src/sonic-host-services/scripts/hostcfgd

```diff
@@ -546,6 +589,135 @@ class AaaCfg(object):
 
         return(new_ipv4_addr, new_ipv6_addr)
 
+    def set_passw_hardening_policies(self, passw_policies):
```

Comment:
I did similar as in other files that this daemon renders.
So I always render the file, but when the feature is disabled, it will render the file with the Linux default values instead the password hardening policies configured

Reply:
In this case the rendered file cannot be an empty file, the reason is:
the name of the file that the feature is rendering is named /etc/pam.d/common-password. And even if you look in a version without the password-hardening feature, this file contained 3 lines:
`
password        [success=1 default=ignore]      pam_unix.so obscure yescrypt
password        requisite                       pam_deny.so
password        required                        pam_permit.so
`

So, the logic is, if the feature is enabled it will add additional lines to support password-hardening, and if the feature is disabled, it will render the file as the 'original' file (same as when the feature does not exist).